### PR TITLE
[Fix] to enable remote config and crashlytics

### PR DESCRIPTION
**Pull request by pinkeshmars**
### Description
Provide a brief overview of what this documentation update is about. Explain what sections or topics are being added or revised.

[Linear ticket](https://linear.app/flutterflow/issue/DEVR-576/remote-config-setting-path-is-changed) and [magic word](https://linear.app/docs/github#link-prs) Fixes DEVR-576  

## Type of change
- [ ] Typo fix 
- [ ] New feature 
- [x] Enhancement to current docs
- [ ] Removed outdated references
- [ ] Update assets

### CODE REVIEW

File: docs/ff-integrations/firebase/crashlytics.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ keywords: [FlutterFlow, Firebase, Crashlytics]
 # Firebase Crashlytics
 [Firebase Crashlytics](https://firebase.google.com/products/crashlytics) is a crash-reporting tool that helps you catch errors. It enables you to troubleshoot the issue by logging the details, such as the exact line number that caused the error, device name, OS version, and time when the crash happened.
 
-To enable Firebase Crashlytics, navigate to **Settings and Integrations** > **Integrations** > **Firebase Crashlytics** and **Enable Firebase Crashlytics**.
+To enable it, navigate to **Settings and Integrations** > **Project Setup** > **Firebase** > Expand the **Crashlytics** section and **Enable Crashlytics**.
```
Comment (PoojaB26) [inline docs/ff-integrations/firebase/crashlytics.md:13]: Keep the "to enable Firebase Crashlytics" for searchability.

File: docs/ff-integrations/firebase/remote-config.md [hunk 2]
```diff
@@ -25,9 +25,9 @@ Follow the steps below to use the Remote Config:
 
 ### 1. Enable Remote Config
 
-To enable it, navigate to **Settings and Integrations** > **Integrations** > **Firebase Remote Config** and **Enable Remote Config**.
+To enable it, navigate to **Settings and Integrations** > **Project Setup** > **Firebase** > Expand the **Remote Config** section and **Enable Remote Config**.
```
Comment (PoojaB26) [inline docs/ff-integrations/firebase/remote-config.md:28]: Same for this,

**Review by PoojaB26 — APPROVED** [review]
Nit changes.

File: docs/ff-integrations/firebase/crashlytics.md (hunk 1 above)
Comment (pinkeshmars) [reply docs/ff-integrations/firebase/crashlytics.md]: Done!

File: docs/ff-integrations/firebase/remote-config.md (hunk 2 above)
Comment (pinkeshmars) [reply docs/ff-integrations/firebase/remote-config.md]: Done!

**Pull request merged**